tools/sailfish/sailfish.xml

```diff
@@ -73,34 +62,51 @@
                 --mates2 ./mate2.$ext
                 --libType "${single_or_paired.orientation}${single_or_paired.strandedness}"
             #end if
-            --output ./
+            --output ./results
             $biasCorrect
+            $gcBiasCorrect
             --threads "\${GALAXY_SLOTS:-4}"
-
-            #if $fldMean:
-                --fldMean $fldMean
+            $dumpEq
+            #if str($gcSizeSamp):
+                --gcSizeSamp str($gcSizeSamp)
```

Comment:
The `str()` is needed only on the `#if` line, where 0 would be considered false. In the second line is breaking the final command line.

Sorry, I should have been clearer.